api/tests/integration/infrastructure/repositories/campaign-participation-repository_test.js

```diff
@@ -353,48 +353,6 @@ describe('Integration | Repository | Campaign Participation', () => {
 
   });
 
-  describe('#findWithUsersPaginated', () => {
```

Comment:
Il te reste un stub à supprimer dans `tests/unit/domain/usecases/find-campaign-participations-related-to-assessment_test.js`
> ligne 16:  const campaignParticipationRepository = { findByAssessmentId: sinon.stub(), **findWithUsersPaginated: sinon.stub()** };

Reply:
en effet